packages/libs/components/src/map/ChartMarker.tsx

```diff
@@ -315,3 +317,13 @@ function chartMarkerSVGIcon(props: ChartMarkerStandaloneProps): {
     sumValuesString,
   };
 }
+
+export function getChartMarkerDependentAxisRange(
+  data: ChartMarkerProps['data'],
+  isLogScale: boolean
+) {
+  return {
+    min: isLogScale ? 0.1 : 0, // matches what we pass into map markers
```

Comment:
0.1 would be fine as a minimum for count-based dependent axes, but I see that in at least one place (`chartMarkerData` for `continuousMarkerPreview` in MapAnalysis.txt) proportion data is being passed.

So something like this would be safer, in case there are positive values smaller than 0.1.

```
min: isLogScale ? Math.min(0.1, ...data.filter(({value}) => value > 0).map(({value}) => value)) : 0,
```

Reply:
In fact, it seems to be an issue with continuous vars...

before:
![image](https://github.com/VEuPathDB/web-monorepo/assets/308639/fc2764e7-eb47-41a4-b7a9-a7a8afc66c96)

after:
![image](https://github.com/VEuPathDB/web-monorepo/assets/308639/9ab8667e-22ba-4019-b61f-a38faa437417)

Hmm, it's better, but something's still not right. It doesn't look roughly like all the map markers:

![image](https://github.com/VEuPathDB/web-monorepo/assets/308639/19c8dd1d-2801-4cf6-ab22-60a07116966b)

Reply:
OK, it seems like the answer is NOT to do `value: d.count / totalCount,` in MapAnalysis.tsx:L601 

It's fine to pass the regular `dataWithCounts` to `ChartMarkerStandalone` and `getChartMarkerDependentAxisRange`

The maths for the proportions doesn't quite work out properly, but I think it's OK because they look kind of right, and a bit different from the count-based log preview markers, which is desirable, to give some UI feedback when the user changes from counts to proportions (in log mode or not).

Let's look at this again when you've had a chance to have another look.

Reply:
To be really strict - the "proportion to counts to sum to proportions" is probably necessary but I'm not sure we need to. Let's see how it looks/behaves.